src/carousel/carousel.ts

```diff
@@ -183,6 +195,10 @@ export class NgbCarousel implements AfterContentChecked,
     }
   }
 
+  showNavigation(): boolean { return !this.hideNavigation || (this.slides != null && this.slides.length > 1); }
+
+  showIndicator(): boolean { return !this.hideIndicator || (this.slides != null && this.slides.length > 1); }
```

Comment:
@wilhelmhb could you please remove the ` this.slides.length > 1` check (and the corresponding tests(s) based on https://github.com/ng-bootstrap/ng-bootstrap/issues/2274#issuecomment-381173512 ?

Reply:
if we have more than one slide, we probably still want to display the navigation item, right ?
Otherwise the user won't see that he/she can slide through the several slides.